2-mandatory/4-people-i-know.js

```diff
@@ -377,14 +377,16 @@ In the above object you can see my friends and the colleagues of my friends.
 First, I want you to find all of my friends who are 35 or older.
 */
 
-let thirtyFiveOrOlder = [];
+let thirtyFiveOrOlder = friends.filter((friend) => friend.age >= 35);
```

Comment:
Great code again! I just want to point out you can destructure this object and do something like

`let thirtyFiveOrOlder = friends.filter(({ age }) => age >= 35)`

Again, both are correct, it is just something to know about :)

Reply:
Oh, I would again, always change this `let` to a `const`. I know I've said it a lot and it's annoying, but I think doing this is 100% the best practise, and I want you to be the best coder possible! ❤️